src/systems/filecoin_mining/storage_mining/mining_cycle.md

```diff
@@ -100,9 +92,9 @@ In this period, the miner can still:
 
 ### Faults
 
-If a miner detects {{<sref storage_faults>}} among their sectors (any sort of storage failure that would prevent them from crafting a PoSt), they should declare these faults with the `DeclareFaults()` method of the Storage Miner Actor. 
+If a miner detects {{<sref storage_faults>}} among their sectors (any sort of storage failure that would prevent them from crafting a PoSt), they should declare these faults with the `DeclareTemporaryFaults()` method of the Storage Miner Actor. 
 
-The miner will be unable to craft valid PoSts over faulty sectors, thereby reducing their chances of winning Election and SurprisePoSts. By declaring a fault, the miner will no longer be challenged on that sector, and will lose power accordingly. The miner can reverse the fault by calling `RecoverFaults()` to add the sector back into the `ProvingSet` and regain power.
+The miner will be unable to craft valid PoSts over faulty sectors, thereby reducing their chances of winning Election and SurprisePoSts. By declaring a fault, the miner will no longer be challenged on that sector, and will lose power accordingly. The miner can specify how long the duration of their TemporaryFault and pay a TemporaryFaultFee.
```

Comment:
[blocking] I am not sure specifying how long the duration of the fault makes sense. These should be unplanned storage outages, and duration should not be predictable. Accordingly, we should just set the fee to the max amount to be conservative, but treat all unplanned outages the same for simplicity.

Reply:
This feels like a policy discussion and agree that it can potentially be problematic when the fault duration is too long. Lets discuss this in sync? cc @jzimmerman

Reply:
Yup. stand by it but let's defer